CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+# 1.4.0 - 2016-08-27
```

Comment:
If you don't mind, I would prefer it if changes to the changelog were handled by me at the time of release.